x-pack/plugins/code/server/utils/composite_source_merger.ts

```diff
@@ -119,8 +119,12 @@ export class LineMapping {
   }
 
   public lineNumber(originLineNumber: number, startAtLine: number = 1) {
-    const n = this.reverseMap.get(originLineNumber) || Number.NaN;
```

Comment:
@spacedragon this is a bug when `this.reverseMap.get(originLineNumber)` returns `0`.

Reply:
good catch